mtags/src/main/scala-2/scala/meta/internal/pc/completions/Completions.scala

```diff
@@ -739,7 +739,7 @@ trait Completions { this: MetalsGlobal =>
   def inferIdentStart(pos: Position, text: String): Int = {
     def fallback: Int = {
       var i = pos.point - 1
-      while (i > 0 && Chars.isIdentifierPart(text.charAt(i))) {
+      while (i >= 0 && Chars.isIdentifierPart(text.charAt(i))) {
```

Comment:
So this was causing issues where if I was trying to do `p@@` to complete package, the lsp edit position start was never 0 here if you were on line 0 and character 1. It was returning an edit with the start and end in the same position causing the edit to append to the previous character instead of replace.